lib/config.js

```diff
@@ -118,6 +118,13 @@ var normalizeConfig = function (config, configFilePath) {
   // normalize urlRoot
   config.urlRoot = normalizeUrlRoot(config.urlRoot)
 
+  config.protocol = config.protocol || 'http:'
+  if (['http:', 'https:'].indexOf(config.protocol) === -1) {
```

Comment:
How about we turn that around; add the `:` when omitted? Reason being that [NodeJS's url.parse](https://nodejs.org/api/url.html#url_url) returns the protocol including `:`, and it is used the same way by `lib/middleware/proxy.js`. If the property includes the colon it is more consistent and allows further use of the `url` class where desired.